packages/cactus-plugin-satp-hermes/src/main/typescript/core/stage-handlers/client/stage1-client-handler.ts

```diff
@@ -64,24 +79,22 @@ export class Stage1ClientHandler {
       sessionData.lastSequenceNumber == undefined //||
       //sessionData.subsequentCalls == undefined ||
       //sessionData.history == undefined ||
-      //sessionData.multipleClaimsAllowed == undefined || 
+      //sessionData.multipleClaimsAllowed == undefined ||
       //sessionData.multipleCancelsAllowed == undefined
     ) {
       throw new Error(`${fnTag}, session data is not correctly initialized`);
     }
 
-    if (!gateway.supportedDltIDs.includes(sessionData.senderGatewayNetworkId)) {
-     throw new Error(
-      `${fnTag}, recipient gateway dlt system is not supported by this gateway`,
-     );
-    }
+    // if (!gateway.supportedDltIDs.includes(sessionData.senderGatewayNetworkId)) {
+    //   throw new Error(
```

Comment:
why is this code deleted?

Reply:
This code was commented out for ease of implementing the new version, will be added back in the next PR